package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "cmcd-validator",
+  "version": "1.0.0",
+  "description": "This is a library created to validate CMCD",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "dev": "nodemon src/index.js"
```

Comment:
Just in case, nodemon is not necessary since Node v18.11.0. You can use `node --watch src/index.js` instead.

But I think that the --watch flag is an experimental feature and nodemon has compatibility with older versions, so maybe is better to keep this how is now.

Reply:
As you said, I think it would be better to add nodemon as a dev dependency since the `--watch` flag is an experimental feature. We can consider this in a future focus.